test/pspm_get_events_test.m

```diff
@@ -133,7 +133,7 @@ function continuous(this)
             this.verifyTrue(length(rimport.data) == length(rimport.markerinfo.value));
             this.verifyTrue(length(rimport.data) == length(d));
             rounded_rimport_data = round(rimport.data*100)/100;
-            this.verifyTrue(all(rounded_rimport_data == rounded_d'));
+            this.verifyTrue(all(rounded_rimport_data == transpose(rounded_d)));
             this.verifyTrue(all(ismember(rimport.markerinfo.value,[1,2])) & ...

```

Comment:
in this file (and maybe others) there is a badly encoded character (see image).
![image](https://user-images.githubusercontent.com/10925826/165335461-59b93401-9d7e-4679-a550-8c44e744b677.png)
It is dangerous to keep them around.
I suggest to run a grep command on the whole project to identify and remove them when appropriate

Reply:
Hi @jbrochar  This issue has been resolved by converting all files into LF and UTF-8 format.